src/rosjava_actionlib/rosjava_actionlib/src/main/java/com/github/ekumen/rosjava_actionlib/ClientStateMachine.java

```diff
@@ -87,7 +89,7 @@ public static String translateState(int state) {
     }
   }
 
-  int latestGoalStatus;
+  public int latestGoalStatus;
```

Comment:
It is generally not a good idea to make an instance variable public just for the sake of running a test, as that violates encapsulation (I know we did that to quickly make the tests pass, but we should look for a workaround). I would suggest either:

- Find another way to write the test.
- If the previous is not possible, define a getter.